CODEOWNERS

```diff
@@ -0,0 +1,36 @@
+# Core
+/ghcide @pepeiborra
+/hls-graph @pepeiborra
+
+# Plugins
+/plugins/hls-alternate-number-format-plugin @drsooch
+/plugins/hls-brittany-plugin @fendor
```

Comment:
Is that an accident? I can't imagine I am the code-owner of hls-brittany-plugin? :O

Reply:
Oh wow: https://github.com/haskell/haskell-language-server/pull/66

I did not remember that

Reply:
It's fine to say "I don't want to be on the hook for it". If we end up with nobody there then we know we need to find someone!

Reply:
I don't mind it, the code is trivial iirc and I did spend some work on it back HIE time, just don't want to take someone's work as my own. The very-original author seems to be @wz1000 according to https://github.com/haskell/haskell-ide-engine/pull/247/files